code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -60,7 +60,57 @@
 		used = !used
 
 	return
-
+//A vial-loaded hypospray. Cartridge-based!
+/obj/item/weapon/reagent_containers/hypospray/vial
+	name = "hypospray mkII"
+	desc = "A new development from DeForest Medical, this new hypospray takes 30-unit vials as the drug supply for easy swapping."

```

Comment:
Let's use a TSC that already exists, please. NT, VM, or ZH are your best options.